soh/src/overlays/actors/ovl_En_Bom_Bowl_Man/z_en_bom_bowl_man.c

```diff
@@ -177,11 +177,16 @@ void EnBomBowMan_CheckBeatenDC(EnBomBowlMan* this, GlobalContext* globalCtx) {
         this->eyeMode = CHU_GIRL_EYES_AWAKE;
         this->blinkTimer = (s16)Rand_ZeroFloat(60.0f) + 20;
 
-        // Check for beaten Dodongo's Cavern if not rando'd
-        // check for bomb bag if rando'd
-        if ((!gSaveContext.n64ddFlag && 
-             !((gSaveContext.eventChkInf[2] & 0x20) || BREG(2))) ||
-             (gSaveContext.n64ddFlag && (INV_CONTENT(ITEM_BOMB) == ITEM_NONE))) {
+        // Check if beaten Dodongo's Cavern if not rando'd
+        // check for bomb bag or bombchus if rando'd depending on whether chus are in logic
+        bool bombchuBowlingClosed;
+        if (gSaveContext.n64ddFlag) {
+            u8 explosive = Randomizer_GetSettingValue(RSK_BOMBCHUS_IN_LOGIC) ? ITEM_BOMBCHU : ITEM_BOMB;
+            bombchuBowlingClosed = (INV_CONTENT(explosive) == ITEM_NONE);
+        } else {
```

Comment:
Shouldn't this check for a bomb bag *and* bombchus when bombchu's are considered in logic?

Reply:
I can double-check, but I’m pretty sure bowling opens only with chus if they’re in logic.

Reply:
Just checked logic; bombchus are the only requirement if they're in logic.
![image](https://user-images.githubusercontent.com/20016345/184800724-cdfc1a22-318d-4cbe-85d1-f555528bea0e.png)